### PR TITLE
Enable fork choice and switch votes, devnet => now, testnet => epoch 63

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
https://github.com/solana-labs/solana/issues/10519
https://github.com/solana-labs/solana/issues/10311

#### Summary of Changes
Enable fork choice + switch votes immediately on devnet, at epoch 63 on testnet

Fixes #

### CODE REVIEW

File: core/src/replay_stage.rs [hunk 1]
```diff
@@ -1755,9 +1755,10 @@ impl ReplayStage {
 
     pub fn get_unlock_heaviest_subtree_fork_choice(operating_mode: OperatingMode) -> Slot {
         match operating_mode {
-            OperatingMode::Development => std::u64::MAX / 2,
+            OperatingMode::Development => 5000,
```
Comment (mvines) [inline core/src/replay_stage.rs:1758]: Why not 0?

Comment (carllin) [reply core/src/replay_stage.rs:1758]: ah I was just following the trend from the Nonce unlock, is devnet restarted from genesis?

Comment (mvines) [reply core/src/replay_stage.rs:1758]: It is persistent so 5000 is effectively as good as 0.  But 0 is more clear IMO.   We probably can set Nonce unlock back to 0 now too

Comment (carllin) [reply core/src/replay_stage.rs]: agreed, changed!

**Comment by mvines** [comment]
What about `get_unlock_switch_vote_slot()`?  When does that get enabled on testnet?

**Comment by carllin** [comment]
@mvines good point, initially I was thinking I would make sure this one was stable before activating the next, but I think they're actually ok to unlock together, and would save us quite some work.

I'll activate them together then!

**Pull request merged**